### ISSUE TITLE
Can't connect on "low" port, connection on "high" port seems ok.

### ISSUE THREAD

**Issue by mostlyflawles** [ilpaolino/homeylocalapi#7]
Since the setup is quite straight forward, I can't see that I've done anything wrong.
I can't get it work, it seems that the port is not opened. I tried default 574 and now 601.

![Screenshot_1](https://github.com/ilpaolino/homeylocalapi/assets/25802524/b55123e8-7d8e-4654-acd5-e83662ce8f6a)
![Screenshot_2](https://github.com/ilpaolino/homeylocalapi/assets/25802524/001632d8-88d4-4266-a641-281d8f87144d)

http://192.168.1.30:601/test

Browser, Postman and telnet does not respond. I get response on port 80 from Homey so I can't see that there any firewall issue.

**Comment by ilpaolino** [comment]
Do you have any error or other feedback when you save the Local API Server Settings?
Have you tried to restart the app?

**Comment by mostlyflawles** [comment]
No negative or positive feedback at all, but if I reopen 'configure' the changes are there.
Yes, restarted the app several times and now also the homey, no progress.

(The webhook app works fine, just for mention to clarify that i'am doing something right =))

**Comment by ilpaolino** [comment]
can you try on an other port 3000 or 5000 for example?

**Comment by mostlyflawles** [comment]
Yes, both those work after I've restarted the app. How come those are working?

**Comment by ilpaolino** [comment]
It's interesting. 
There is probably a Homey block on a "low" http port range.
What version of homey are you using? the 2019 or the new one?

I'm trying to investigate to understand why this happens.

**Comment by mostlyflawles** [comment]
Model Homey (Early 2019)
Version 10.0.2
